app/controllers/account_controller.rb

```diff
@@ -25,8 +25,8 @@ def account_params
       if @account.organization?
         params.require(:account).permit(:phone_number, :email_on_comment, :email_on_comment_reply, :newsletter, organization_attributes: [:name, :responsible_name])
       else
-        params.require(:account).permit(:username, :public_activity, :email_on_comment, :email_on_comment_reply, :email_on_direct_message, :email_digest, :newsletter, :official_position_badge)
+        params.require(:account).permit(:username, :public_activity, :public_interest, :email_on_comment, :email_on_comment_reply, :email_on_direct_message, :email_digest, :newsletter, :official_position_badge)
```

Comment:
+1 Yes it sounds more descriptive, right? Feature is "User can mark his interests as public." so plural makes sense.